templates/libreswan-host-to-host.conf.j2

```diff
@@ -6,7 +6,7 @@
 {%         if tunnel.hosts[host] is mapping and 'hostname' in tunnel.hosts[host] %}
 {%           set host = tunnel.hosts[host].hostname %}
 {%         endif %}
-conn {{ tunnel.name | d(host + '-to-' +  item)}}
+conn {{ (tunnel.name | d('') | length > 0) | ternary(tunnel.name + '-','') }}{{ host }}-to-{{ item }}
```

Comment:
Do we need a unique prefix if name is not provided?

Reply:
I don't think we need the prefix if `name` is not provided. We are using the `host_x-to-host_y` convention which should eliminate the possibility of duplicates. The code omits the prefix when no `name` is specified.